EasyCommands/CommandParsers/ParameterParsers.cs

```diff
@@ -449,36 +450,41 @@ partial class Program {
             return commandParameters;
         }
 
-        public List<Token> ParseTokens(String commandString) => (String.IsNullOrWhiteSpace(commandString) || commandString.Trim().StartsWith("#")) ? NewList<Token>() :
-            ParseSurrounded(commandString, "`\'\"",
+        delegate IEnumerable<String> Pass(String s);
+        public List<Token> Tokenize(String commandString) {
+            Pass thirdPass = v => SeperatorPass(v, thirdPassTokens);
+            Pass secondPass = v => SeperatorPass(v, secondPassTokens, w => PrimitivePass(w, thirdPass));
+            Pass firstPass = v => SeperatorPass(v, firstPassTokens, w => PrimitivePass(w, secondPass));
+
+            return (String.IsNullOrWhiteSpace(commandString) || commandString.Trim().StartsWith("#"))
+            ? NewList<Token>()
+            : TokenizeEnclosed(commandString, "`\'\"",
                 u => u.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries)
-                    .SelectMany(ParseSeparateTokens)
+                    .SelectMany(v => firstPass(v))
```

Comment:
The chaining of all the passes can be done as a (very long) oneliner directly inside the `SelectMany(...)` but that is unreadable.

Reply:
Definitely adds a few chars, but the readability is nice.  Can always revisit later